src/content/en/updates/2017/06/headless-karma-mocha-chai.md

```diff
@@ -0,0 +1,137 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Testing on Headless Chrome with Karma/Mocha/Chai"
+
+{# wf_updated_on: 2017-06-13 #}
+{# wf_published_on: 2017-06-13 #}
+
+{# wf_tags: headless,testing,karma,mocha,chai #}
+{# wf_featured_image: /web/updates/images/generic/headless-chrome.png #}
+{# wf_featured_snippet: How to setup Karma, Mocha, and Chai to run automated tests on Headless Chrome. #}
+
+# Testing on Headless Chrome with Karma/Mocha/Chai {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+Karma, Mocha, Chai, Headless Chrome, oh my!
+
+If you want to run automated tests using Headless Chrome, look no further! This article will get you
+all setup using Karma as a runner and Mocha/Chai for authoring tests.
+
+**What are these things?**
+
+[Karma](https://karma-runner.github.io) is a testing harness that works with
+any of the the most popular testing frameworks ([Jasmine](https://jasmine.github.io/), [Mocha](https://mochajs.org/), [QUnit](https://qunitjs.com/)).
+
+[Chai](http://chaijs.com/) is an assertion library that works with Node and in the browser.
+We need the latter.
+
+[Headless Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome) is a way to run the Chrome browser in a headless environment without the full browser UI. It's perfect for CI environment and automated testing.
```

Comment:
Also maybe something about why headless is better than other existing options. Which is that it gives you more accuracy in your automated testing of how a site behaves on Google Chrome, since it's a closer approximation to Google Chrome than existing solutions (right?)